Libplanet.Net/Transports/NetMQTransport.cs

```diff
@@ -599,14 +599,37 @@ private void ReceiveMessage(object sender, NetMQSocketEventArgs e)
                         {
                             try
                             {
+                                Guid reqId;
```

Comment:
I think that `reqId` will be used only for logging purposes, so turning it to`string` and parse like as below would be enough.

```csharp
string requestId;
try
{
    requestId =  new Guid(copied[0].ToByteArray()).ToString();
}
catch (ArgumentException)
{
    requestId = "unknown";
}
```